cmd/bootstrap/build/version.go

```diff
@@ -41,3 +41,11 @@ func init() {
 		commit = undefined
 	}
 }
+
+func SetSemver(sv string) {
```

Comment:
Two setters to make testing easy (without using ldflags)

Reply:
I would much rather simplify testing another way than by adding these setters. Having these makes it too easy to set the wrong version globally.

Instead, what about injecting a function to `PingService` to get the version? It is likely we will want to take a similar approach anyway to include the block height.